x-pack/plugins/maps/public/components/global_filter_checkbox.tsx

```diff
@@ -21,13 +22,21 @@ export function GlobalFilterCheckbox({ applyGlobalQuery, label, setApplyGlobalQu
 
   return (
     <EuiFormRow display="columnCompressedSwitch">
-      <EuiSwitch
-        label={label}
-        checked={applyGlobalQuery}
-        onChange={onApplyGlobalQueryChange}
-        data-test-subj="mapLayerPanelApplyGlobalQueryCheckbox"
-        compressed
-      />
+      <EuiToolTip
+        position="top"
+        content={i18n.translate('xpack.maps.filterEditor.applyGlobalFilterHelp', {
+          defaultMessage:
+            'Whether the query and filters from the global query bar should be applied to the layer.',
```

Comment:
https://www.elastic.co/guide/en/kibana/current/kibana-concepts-analysts.html#kibana-concepts-searching-your-data defines the terms as "semi-structured search" and "extra filters".

How about "When enabled, semi-structured search and extra filters from global search bar are applied to this layer". 

cc @gchaps

Reply:
Actually, How about 

"When enabled, this layer is narrowed by search bar"

This matches the text in the legend that says "Results narrowed by search bar"